lib/charms/prometheus_k8s/v0/prometheus_scrape.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2023 Canonical Ltd.
+# Copyright 2021 Canonical Ltd.
```

Comment:
That's an odd change :)

Reply:
Maybe they have a [DeLorean](https://en.wikipedia.org/wiki/DeLorean_time_machine) ;-)